[PATCH] Support Python 2

Changes for Python 2 compatibility.

Inappropriate for submission due to EOL.

Upstream-Status: Inappropriate [other]
Signed-off-by: offa

---
diff --git a/setup.py b/setup.py
index 212c0c1..f89efef 100644
--- a/setup.py
+++ b/setup.py
@@ -1,7 +1,7 @@
 """Setuptools install script for pytest-timeout."""
 from setuptools import setup
 
-with open("README.rst", encoding="utf-8") as f:
+with open("README.rst") as f:
     long_description = f.read()
 
 
